packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Remove unnecessary imports.
```

Comment:
Nit: This should be "Removes" throughout the PR; see the CHANGELOG style guide linked from the PR checklist.